packages/messaging/ios/RNFBMessaging/RNFBMessagingModule.h

```diff
@@ -20,5 +20,5 @@
 
 
 @interface RNFBMessagingModule : NSObject <RCTBridgeModule>
-
++ (NSMutableDictionary *) addCustomPropsToUserProps: (NSDictionary *__nullable)userProps withLaunchOptions: (NSDictionary *__nullable) launchOptions;
```

Comment:
```suggestion
+ (NSDictionary *) addCustomPropsToUserProps: (NSDictionary *__nullable)userProps withLaunchOptions: (NSDictionary *__nullable) launchOptions;
```

Doesn't need to be a mutable dict since internally we call `mutableCopy`.